app/sign.js

```diff
@@ -131,8 +131,11 @@ const getBackupSigningKey = function(key, expectedXpub) {
 }
 
 const handleSignUtxo = function(recoveryRequest, key, skipConfirm) {
-  const network = utxoNetworks[recoveryRequest.coin];
-  const decimals = coinDecimals[recoveryRequest.coin];
+  
+  // const network = utxoNetworks[recoveryRequest.coin];
+  // const decimals = coinDecimals[recoveryRequest.coin];
+  const network = statics.coins.get[recoveryRequest.coin].network;
```

Comment:
Might be worth adding an alias `getCoinConfig(coin)` to avoid having to write `statics.coins.get[x]` over and over

Reply:
Good point, but consider it only used 4 times in this file so I didn't bother

Reply:
Added. Not sure if this is the optimal way tho.